app/_data/docs_nav_gateway_3.0.x.yml

```diff
@@ -11,504 +11,529 @@ items:
+            items:
+              - text: Easy Theme editing
+                url: /kong-enterprise/developer-portal/customize/easy-theme-editing
+              - text: Migrating Templates Between Workspaces
+                url: /kong-enterprise/developer-portal/customize/migrating-templates
+              - text: Markdown Rendering Module
+                url: /kong-enterprise/developer-portal/customize/markdown-extended
+              - text: Customizing Portal Emails
+                url: /kong-enterprise/developer-portal/customize/emails
+              - text: Adding and Using JavaScript Assets
+                url: /kong-enterprise/developer-portal/customize/adding-javascript-assets
+              - text: Single Page App in Dev Portal
+                url: /kong-enterprise/developer-portal/customize/single-page-app
+              - text: Alternate OpenAPI Renderer
+                url: /kong-enterprise/developer-portal/customize/alternate-openapi-renderer
+          - text: SMTP
+            url: /kong-enterprise/developer-portal/smtp
+          - text: Workspaces
+            url: /kong-enterprise/developer-portal/workspaces
+          - text: Helpers CLI
+            url: /kong-enterprise/developer-portal/cli
+          - text: Portal API
+            url: /kong-enterprise/developer-portal/portal-api         
+      - text: Audit Logging
+        url: /kong-enterprise/audit-log
+      - text: Keyring and Data Encryption
+        url: /kong-enterprise/db-encryption
+  - title: Kong Manager
+    icon: /assets/images/icons/documentation/icn-references-color.svg
+    items: 
+      - text: Overview
+        url: /kong-manager/
+      - text: Enable Kong Manager
+        url: /kong-manager/enable-km
+      - text: Get Started with Kong Manager
         items:
-          - text: Easy Theme Editing
-            url: /developer-portal/theme-customization/easy-theme-editing
-          - text: Migrating Templates Between Workspaces
-            url: /developer-portal/theme-customization/migrating-templates
-          - text: Markdown Rendering Module
-            url: /developer-portal/theme-customization/markdown-extended
-          - text: Customizing Portal Emails
-            url: /developer-portal/theme-customization/emails
-          - text: Adding and Using JavaScript Assets
-            url: /developer-portal/theme-customization/adding-javascript-assets
-          - text: Single Page App in Dev Portal
-            url: /developer-portal/theme-customization/single-page-app
-          - text: Alternate OpenAPI Renderer
-            url: /developer-portal/theme-customization/alternate-openapi-renderer
-      - text: Helpers CLI
-        url: /developer-portal/helpers/cli
-
-  - title: Monitor
-    icon: /assets/images/icons/documentation/icn-vitals.svg
-    items:
-      - text: Kong Vitals
-        url: /vitals
+          - text: Services and Routes
+            url: /kong-manager/get-started/services-routes
+          - text: Rate Limiting
```

Comment:
For some reason, this page is redirecting to the "Services and Routes" page above it for me.